### ISSUE TITLE
0.1.3 seems not deployed to the maven repo.

### ISSUE THREAD

**Issue by cosmir17** [wolfendale/scalacheck-gen-regexp#9]
Hi,

Thank you for this wonderful library.

0.1.3 was released 22 days ago but seems not deployed to the maven repo.

https://mvnrepository.com/artifact/wolfendale/scalacheck-gen-regexp

Hope this helps,
Sean

**Comment by cosmir17** [comment]
I found a different dependency for 0.1.3
https://mvnrepository.com/artifact/io.github.wolfendale/scalacheck-gen-regexp

Can you  mentioned in the github repo readme, please? so that people won't get confused as I was.

As soon as I have a reply (either yes or no), I will close this ticket.

**Comment by wolfendale** [comment]
Hi @cosmir17, Sorry for the confusion!

The library used to be just under the `wolfendale` groupid. I'd never really deployed a library anywhere before and it seemed as good as anything else. This was fine when it was hosted on bintray but unfortunately that service has been stopped by jFrog so I had to migrate to maven central (which I should have done in the first place!) Maven central requires proof of ownership for a registered groupid and `wolfendale` wasn't valid so I updated the groupid to `io.github.wolfendale` and deployed there.

Unfortunately this means anyone migrating from < 1.3 to 1.3 will need to change both the version and the groupId. However it also means people don't need to add a custom bintray resolver that will no longer work.

I updated the README to remove the old resolver and use the correct groupId when I migrated so hopefully this hasn't caused too much trouble. I can't see anything else that needs updating, which part of the README is unclear?
